Lib/fontmake/font_project.py

```diff
@@ -25,19 +25,12 @@
 from cu2qu.rf import fonts_to_quadratic
 from fontTools import subset
 from glyphs2ufo.glyphslib import build_masters, build_instances
+from mutatorMath.ufo import build as build_designspace
+from robofab.world import OpenFont
```

Comment:
no, not yet. If have some ideas as to how to adapt the pen protocol to the *Collection classes in `cu2qu.rf` module, but don't have anything to show yet. Suggestions are welcome.